ui/partials/perf/comparesubtestctrl.html

```diff
@@ -7,6 +7,16 @@
   <div id="error" ng-if="!dataLoading && errors.length">
     <compare-error errors="errors" original-revision="originalRevision" original-project="originalProject" new-revision="newRevision" new-project="newProject"></compare-error>
   </div>
+    <div class="alert alert-warning" role="alert" ng-if="testsTooVariable.length > 1 && filterOptions.showOnlyNoise">
```

Comment:
I don't think you need the `> 1` here.

Reply:
the 1st entry in the array is the hacky title row, so I want to ignore that if there are no tests we needed to ignore- the rest of the suggestions are great and make sense.